Add support for Akeneo 3.0 (#18)
flagbit/akeneo-table-attribute-bundle: CHANGELOG-3.0.md

# 3.0.0

- Add support for Akeneo 3.0.0.

## BC breaks

> - `Flagbit\Bundle\TableAttributeBundle\Normalizer\StructuredAttributeOptionNormalizer` was deleted and that `Flagbit\Bundle\TableAttributeBundle\Normalizer\AttributeOptionNormalizer` is taking its job in its service.

Reviewer: ```suggestion
- `Flagbit\Bundle\TableAttributeBundle\Normalizer\StructuredAttributeOptionNormalizer` was deleted. `Flagbit\Bundle\TableAttributeBundle\Normalizer\AttributeOptionNormalizer` is now used for its service.
```
